src/asami/storage.cljc

```diff
@@ -20,6 +20,7 @@
 (defprotocol Database
   (as-of [this t] "Retrieves a database as of a given moment, inclusive")
   (as-of-t [this] "Returns the t point for a database")
+  (as-of-time [this] "Returns the timestamp saved for a database")
```

Comment:
I think "associated with" instead of "saved for" may be a better term here

Reply:
Sure, I'll reword :)

Reply:
Done.

I've rebased main because I saw that you've merged it into this PR. I figured doing it this was is neater.

Thanks